packages/twenty-front/src/modules/ui/utilities/state/component-state/hooks/useExtractedComponentState.ts

```diff
@@ -0,0 +1,27 @@
+import { useAvailableScopeIdOrThrow } from '@/ui/utilities/recoil-scope/scopes-internal/hooks/useAvailableScopeId';
+import { getScopeIdOrUndefinedFromComponentId } from '@/ui/utilities/recoil-scope/utils/getScopeIdOrUndefinedFromComponentId';
+import { RecoilComponentState } from '@/ui/utilities/state/component-state/types/RecoilComponentState';
+
+export const useExtractedComponentState = <Value>(
```

Comment:
I don't think we need this hook, let's use useRecoilComponentState that you have introduced below

Reply:
We need it for useRecoilCallback